python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -382,3 +383,103 @@ def measureForcedSources(self, diaSources, science, wcs):
                           "ip_diffim_forced_PsfFlux_flag_edge", True)
         for diaSource, forcedSource in zip(diaSources, forcedSources):
             diaSource.assign(forcedSource, mapper)
+
+
+class DetectAndMeasureScoreConnections(DetectAndMeasureConnections):
+    scoreExposure = pipeBase.connectionTypes.Input(
+        doc="Output likelihood score image",
+        dimensions=("instrument", "visit", "detector"),
+        storageClass="ExposureF",
+        name="{fakesType}{coaddName}Diff_scoreExp",
+    )
+
+
+class DetectAndMeasureScoreConfig(DetectAndMeasureConfig,
+                                  pipelineConnections=DetectAndMeasureScoreConnections):
```

Comment:
Do you even need this class, if it's just `DetectAndMeasureConfig`? Until you add/override things in it, might as well just use `DetectAndMeasureConfig` as the `ConfigClass` for the new Task.

Reply:
Yes, it's needed to set the connections.